ui-components/src/page_components/PartnersHeader.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo, useContext } from "react"
```

Comment:
@emilyjablonski component name sounds good to me.

I defined this component in a component library, because partners header should use another `font-family` than public site. I had to define new `.scss` file and CSS variable to change a font-size following new theming pattern and I think that's possible only in the component lib.

I don't want duplicate logic that is inside the `PartnersHeader` component so that's why I put this component in the `page_components` directory - this component should be only for partners purposes :)